dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -83,6 +83,8 @@ Future<void> _dummyAsyncVoidCallback() async {}
 @sealed
 class Runner {
   /// Creates a runner for the [recorder].
+  ///
+  /// All arguments must not be null.
```

Comment:
Something is odd about this PR, it seems to add the "must not be null" comment in a couple of places??

Reply:
Yes, I messed up when I was splitting this out into a separate PR.  Should be fixed now.